performance/tests/Alert.test.js

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
`testMultiple` should be removed from global varaible list since now we use `newPageMultiple`
![image](https://user-images.githubusercontent.com/63335823/195342901-5ba7ec1c-1008-4c32-b07a-709f1f7f5666.png)

This fix need to be applied for each test